### PR TITLE
Added Dicos wikidata & wikipedia

### PR DESCRIPTION
**Pull request by yelaD**

### CODE REVIEW

File: data/brands/amenity/fast_food.json [hunk 1]
```diff
@@ -6195,12 +6207,18 @@
...
       "tags": {
         "amenity": "fast_food",
         "brand": "德克士",
+        "brand:en": "Dicos",
+        "brand:wikidata": "Q5273883",
+        "brand:wikipedia": "zh:德克士",
+        "brand:zh": "德克士",
         "name": "德克士",
+        "name:en": "Dicos",
+        "name:zn": "德克士",
```
Comment (Identitaet) [inline data/brands/amenity/fast_food.json:6221]: Shouldn't this be zh

Comment (bhousel) [inline data/brands/amenity/fast_food.json:6221]:
```suggestion
        "name:zh": "德克士",
```

**Comment by bhousel** [comment]
welcome @yelaD 👋 
I fixed the `name:zh` issue and merged..

File: data/brands/amenity/fast_food.json (hunk 1 above)
Comment (yelaD) [reply data/brands/amenity/fast_food.json]: > Shouldn't this be zh

Oops, honest mistake. Sorry about that.

**Comment by yelaD** [comment]
> welcome @yelaD wave

Thanks for having me.

**Pull request merged**